### PR TITLE
changed the version retrieval method

### PR DESCRIPTION
**Pull request by kevupton**
references issue:

https://github.com/aloha/laravel-twilio/issues/133

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 /.php_cs.cache
 /composer.lock
 /vendor
+.idea
```
Comment (hannesvdvreken) [inline .gitignore]: Remove this. This belongs in your personal global gitignore.

File: src/Support/Laravel/ServiceProvider.php [hunk 2]
```diff
@@ -36,12 +35,20 @@ protected function provider()
...
+        if ($this->is_laravel() && version_compare($this->app->version(), '5.0', '<')) {
             $this->provider = new L4ServiceProvider($this->app);
         } else {
             $this->provider = new L5ServiceProvider($this->app);
         }
 
         return $this->provider;
     }
+
+    private function is_lumen() {
```
Comment (hannesvdvreken) [inline src/Support/Laravel/ServiceProvider.php]: seems to be unused

**Comment by TheDeadCode** [comment]
Any plans to merge?

**Comment by hannesvdvreken** [comment]
@TheDeadCode @kevupton 

This doesn't work because

https://github.com/laravel/framework/blob/5.5/src/Illuminate/Foundation/Application.php#L163
and
https://github.com/laravel/lumen-framework/blob/5.5/src/Application.php#L134

are not the same.

Also
- `class_exists()` isn't a very good differentiator. Butter to check if `$this->app instanceof`.
- Would you use Laravel 5 service provider for every version of Lumen? How is that guaranteed to work?

**Comment by hannesvdvreken** [comment]
Just write your own SP when in Lumen

**Pull request closed without merging**